bulbs/special_coverage/utils.py

```diff
@@ -0,0 +1,37 @@
+"""Common utilities for special coverage behaviors."""
+from django.conf import settings
+from django.utils import timezone
+
+from bulbs.content.filters import Published
+from .models import SpecialCoverage
+from .search import SearchParty
+
+
+def get_sponsored_special_coverages():
+    """:returns: Django query for all active special coverages with active campaigns."""
+    now = timezone.now()
+    return SpecialCoverage.objects.filter(
+        campaign__isnull=False,
+        campaign__start_date__lte=now,
+        campaign__end_date__gte=now,
+        start_date__lte=now,
+        end_date__gte=now
```

Comment:
We should standardize on our "greater/less than" and "greater/less"

I think it's most intuitive to have "start" be inclusive and "end" be exclusive -- so  `start_date <= now < end`, which makes it easy to have adjacent ranges that don't overlap.